src/piidigger/classes.py

```diff
@@ -198,17 +191,11 @@ def setMaxProcs(self, procs):
 class ProcessManager:
     def __init__(self, 
                  name: str,
-                 logQ: mp.Queue,
-                 logLevel: str):
+                 logManager: LogManager,):
         
         self.processes: list = []
         self.name = name
-        self.logger=logging.getLogger(name)
```

Comment:
This is the general pattern that is repeated everywhere else...

1. Remove the 4 or so commands that instantiated and configured a new logger instance as all of these have now been incoporated into LogManager.getLogger().  
2. Instead, call the logmanager,getLogger() method, which will return a fully-configured logger instance.